test/testtype.cpp

```diff
@@ -495,6 +495,46 @@ class TestType : public TestFixture {
               "  return 1234.5;\n"
               "}", settingsDefault);
         ASSERT_EQUALS("[test.cpp:2]: (error) Undefined behaviour: float () to integer conversion overflow.\n", removeFloat(errout_str()));
+
+        const Settings settingsWithStdint = settingsBuilder().library("std.cfg").build();
```

Comment:
I don't think loading std.cfg is required. Can you cast to "unsigned short" instead of uint16_t? That should save some cpu cycles.